code/modules/mob/living/carbon/human/human.dm

```diff
@@ -883,17 +881,6 @@
 
 	return species.name
 
-/mob/living/carbon/human/proc/play_xylophone()
```

Comment:
no, this is a blocker, this proc must remain.

Reply:
https://youtu.be/XVQOhIVcMj0

Reply:
This is a good feature. It should stay, even though very spooky.

Reply:
Poe's law

Reply:
Talking more seriously, keep it there. It's funny enough.

Reply:
It's not even usable and relies on non-existant dna code

Reply:
It can be invoked by admins. /tg/ has skeletons.
This is a little gem, nothing too evil.

Reply:
Nothing uses it and you know our stance on that. So far it has been used exactly 0 times, procs that aren't used anywhere but are just for admins are very unlikely to be ever used.